app/templates/documents.rb

```diff
@@ -152,6 +160,12 @@ def tokenizer(json)
         json.delimiter "."
         json.reverse true
       end
+      json.domain_minus_ext_tokenizer do
+        json.type "pattern"
+        json.pattern "\\b(\\S+)\\.com\\b"
```

Comment:
@jamesisaacs , the last time we talked about this, you were going to make the tokenizer more flexible. This will work for "amazon.com", but not "www.amazon.com", "www.amazon.co.uk", "amazon.gov", etc.

Reply:
@MothOnMars I think we talked about splitting on the `.` but I decided to not do that because it adds too much noise. Adding additional TLDs and stripping `www` seem like good ideas to me. I don't think we need to worry about separating subdomain parts. I'll ask Dawn for her preference in in the PT story.